docs/how-to/topics/file-management.rst

```diff
@@ -0,0 +1,206 @@
+
+For this guide, we're going to use Briefcase to package our application. However, the same path-related issues outlined here exist regardless of what tool you use to deploy your code, including Setuptools, py2app, PyInstaller, and so on.
+
+File access
+===========
+
+If you've worked with accessing files in Python, you're probably used to being able to read from a file in the same directory as your Python script using something like the following:
+
+.. code-block:: python
+
+    with open("file.txt") as f:
+        f.read()
+
+This is an example of a relative path, meaning Python is expecting ``file.txt`` to be in a specific location relative to the Python program itself. Python defaults to looking in the *current working directory*, that is, the directory from which the program is being run. You are able to use relative paths to point to a file based on that location. However, this means the same relative path will point to a different location if the program is run from a different directory.
+
+Absolute paths are the full path to a location, constructed using the root directory of the file system as the anchor (``/`` on macOS and Linux, and ``C:\`` on Windows). They do not rely on the location of the Python program or app, and they are valid from anywhere in the file system. At runtime, a relative path is resolved to an absolute path, constructed using the current working directory as the anchor.
+
+Relative paths might work with simple Toga scripts. However, when you get into packaging applications, things change. When you double-click on an icon to run an app, what directory is considered the current working directory? Is it the directory containing the app bundle? Is it the directory in the app bundle that holds the executable? There isn't a solid answer. The question we really need to answer is, how can we reliably tell an app where to look for file content?
+
+If we can't rely on the current working directory to construct relative paths, what *can* we use? We need to use absolute paths.
+
+Absolute paths
+==============
+
+There are multiple ways to construct absolute paths in Python.
+
+The most basic method is to spell out the entire path. For example, on macOS or Unix, to access ``file.txt`` in a directory called ``file_directory`` in your home directory, you could use ``~/file_directory/file.txt``.
+
+In any Python file, ``__file__`` resolves to the absolute path to the location of file being executed, provided as a string. We can use this to construct paths; for example, ``Path(__file__).parent`` is the parent directory of the currently running Python program.
```

Comment:
Might want a lead in here like "Python also provides some tools to build absolute paths based on properties of the running python code."